docs/workflow/building/libraries/webassembly-instructions.md

```diff
@@ -4,7 +4,10 @@
 
 If you haven't already done so, please read [this document](../../README.md#Build_Requirements) to understand the build requirements for your operating system.
 
-The Emscripten SDK (emsdk) needs to be installed.  Follow the installation guide [here](https://emscripten.org/docs/getting_started/downloads.html#sdk-download-and-install) or run `make -C src/mono/wasm provision-wasm` to install emsdk into `src/mono/wasm/emsdk`.
+The [expected version](..\..\..\..\src\mono\wasm\emscripten-version.txt) of Emscripten SDK (emsdk) needs to be installed.
+* Run `make -C src/mono/wasm provision-wasm` to install emsdk into `src/mono/wasm/emsdk`.
+* Alternatively follow the [installation guide](https://emscripten.org/docs/getting_started/downloads.html#sdk-download-and-install).
+Do not install `latest` but rather specific version e.g. `./emsdk install 2.0.12`
```

Comment:
You might mention https://github.com/dotnet/runtime/blob/main/src/mono/wasm/emscripten-version.txt here, it contains the current version used to build mono webassembly.

Reply:
improved